<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gTdsParams">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>401</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>TDS Pulse Parameters for periodic scanning:</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>371</width>
       <height>171</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblTdsStart">
           <property name="text">
            <string>start (ps):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditTdsStart"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="lblTdsEnd">
           <property name="text">
            <string>end (ps):  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditTdsEnd"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblTdsAvgs">
           <property name="text">
            <string>avgs:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditTdsAvgs"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblTdsFrames">
           <property name="text">
            <string>frames:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditFrames"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblTdsInterval">
           <property name="text">
            <string>interval:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lEditInterval"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtnCreateGIF">
         <property name="text">
          <string>create GIF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioBtnKeepGIFSource">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>keep GIF source</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>THEA TIMELAPSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ScanControl Status: Idle</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Timelapse progress:</string>
    </property>
    <widget class="QProgressBar" name="progTlapse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblFrameCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame count: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Scan Progress</string>
    </property>
    <widget class="QProgressBar" name="progAvg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
